MinPlatformPkg/Test/Library/TestPointCheckLib/DxeCheckMtrr.c

```diff
@@ -0,0 +1,411 @@
+/** @file
+
+Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <PiPei.h>

```

Comment:
`PiPei.h` should not be included in a file targeting DXE.

Reply:
Fixed.